terraform/applications/discourse/users.tf

```diff
@@ -0,0 +1,115 @@
+####################
+#      Users       #
+####################
+
+// Grant access to users via MAWS (prod only)
+module "discourse_developers" {
+  create_role  = var.environment == "prod" ? "1" : "0"
+  source       = "github.com/mozilla-it/terraform-modules//aws/maws-roles?ref=master"
+  role_name    = "maws-discourse-developers"
+  role_mapping = ["aws_discourse_dev"]
+  policy_arn   = var.environment == "prod" ? [aws_iam_policy.developers[0].arn] : [""]
+}
+
+resource "aws_iam_policy" "developers" {
+  count  = var.environment == "prod" ? "1" : "0"
+  name   = "discourseDevelopersPolicy"
+  path   = "/"
+  policy = data.aws_iam_policy_document.developers.json
+}
+
+data "aws_iam_policy_document" "developers" {
+
+  statement {
+    sid       = "DiscourseEks"
+    actions   = ["eks:DescribeCluster"]
+    resources = ["*"]
+  }
+
+  statement {
+    sid     = "DiscourseCodebuild"
+    actions = ["codebuild:*"]
+    resources = [
+      "arn:aws:codebuild:us-west-2:783633885093:project/discourse-stage",
```

Comment:
You could interpolate the account id for these and the following rules. I see this is directly imported so no big deal, nothing we need to change now.

Reply:
that's true, will consider for the future since i think these users resources will change in the next month or two anyways